modules/covid_vaccine/spec/request/covid_vaccine/v0/registration_request_spec.rb

```diff
@@ -52,7 +52,14 @@
     end
 
     context 'when encountering an Internal Server Error' do
-      let(:registration_attributes) { { date_vaccine_reeceived: '' } }
+      let(:registration_attributes) do
+        {
+          vaccine_interest: 'yes',
+          email: 'jane.doe@email.com',
+          zip_code: '94402',
+          date_vaccine_reeceived: ''
```

Comment:
Not sure if it matters here but FE is no longer collecting the date a vaccine was received.

Reply:
yeah, I think that's just for a dumb spec to test what happens when a 500 error returns -- could be any misspelled field that we send to VEText.